jujugui/static/gui/src/app/store/env/web-sandbox.js

```diff
@@ -29,12 +29,15 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 YUI.add('juju-env-web-sandbox', function(Y) {
 
   var module = Y.namespace('juju.environments.web');
-  var localCharmExpression = /\/juju-core\/charms\?series=(\w+)/;
+  var localCharmExpression = new RegExp(
+      /\/juju-core\/model\/[A-Za-z0-9-]+\/charms\?/.source +
```

Comment:
Hm. I don't love littering our code with the disable comment, and this was actually a bit easier to read to me since it's broken up into path and querystring components.